Doc/whatsnew/3.2.rst

```diff
@@ -2593,10 +2593,12 @@ Changes to Python's build process and to the C API include:
   longer used and it had never been documented (:issue:`8837`).
 
 There were a number of other small changes to the C-API.  See the
-:source:`Misc/NEWS` file for a complete list.
+Misc/NEWS <https://github.com/python/cpython/blob/v3.2.6/Misc/NEWS>`_
```

Comment:
```suggestion
`Misc/NEWS <https://github.com/python/cpython/blob/v3.2.6/Misc/NEWS>`_
```

For some reason, gh-94078 backport is the only one that caught this typo.

Reply:
Possibly because it was using the old docs checker script as opposed to Sphinx-Lint. Could you verify whether the latest Sphinx-Lint 0.6.6 catches this case, and if not, could you report it upstream? Thanks!

Reply:
Also, it really isn't great that we're regressing from using a dedicated role to hardcoded links...but hopefully the new suite of roles we're developing for this will replace this.

Reply:
> Could you verify whether the latest Sphinx-Lint 0.6.6 catches this case [...]

I ran sphinx-lint 0.6.7 and it caught the typo successfully:

    C:\Users\oleg\Documents\dev\notmine\cpython>sphinx-lint Doc/whatsnew/3.2.rst
    Doc/whatsnew/3.2.rst:54: missing backtick before hyperlink reference: 'Misc/NEWS <https://github.com/python/cpython/blob/v3.2.6/Misc/NEWS>`_'. (hyperlink-reference-missing-backtick)

So I hope that adoption of sphinx-lint will eliminate such sporadic discoveries.